test/cases/hmr/expected/webpack-5/main.js

```diff
@@ -1,7 +1,3755 @@
 /******/ (() => { // webpackBootstrap
-/******/ 	/************************************************************************/
+/******/ 	var __webpack_modules__ = ([
```

Comment:
I don't think snapshotting to complete output file is a good idea. Neither for 4 or 5.